pkgs/os-specific/linux/trusting-trust/stage0-posix/mescc-tools/build.kaem

```diff
@@ -0,0 +1,184 @@
+# This is a modified version of stage0-posix/x86/mescc-tools-full-kaem.kaem
+# https://github.com/oriansj/stage0-posix-x86/blob/56e6b8df3e95f4bc04f8b420a4cd8c82c70b9efa/mescc-tools-full-kaem.kaem
```

Comment:
The original version has copyright headers and license information.  If we want to distribute a work based on it, we have to keep those, or we're violating the license.  If you've copied any other files, please make sure those licenses are being followed as well.